### PR TITLE
Fix doc for no associativity

### PR DESCRIPTION
**Pull request by JasonGross**
no associative -> no associativity

Also remove some 'a's and 'the's and make a note that this is for parsing
(There is a difference between left associativity and no associativity for printing)

**Kind:** documentation / bug fix.

### CODE REVIEW

File: doc/sphinx/user-extensions/syntax-extensions.rst [hunk 1]
```diff
@@ -1455,7 +1455,9 @@ tactic language. Tactic notations obey the following syntax:
 .. [#and_or_levels] which are the levels effectively chosen in the current
    implementation of Coq
 
-.. [#no_associativity] Coq accepts notations declared as ``no associative`` but the parser on
-   which Coq is built, namely Camlp4, currently does not implement the
-   ``no associativity`` and replaces it by a ``left associativity``; hence it is
-   the same for Coq: ``no associativity`` is in fact ``left associativity``.
+.. [#no_associativity] Coq accepts notations declared as ``no
+   associativity`` but the parser on which Coq is built, namely
+   Camlp4, currently does not implement ``no associativity`` and
```
Comment (Zimmi48) [inline doc/sphinx/user-extensions/syntax-extensions.rst]: Camlp4 -> Camlp5

**Review by Zimmi48 — COMMENTED** [review]
I don't see any "a" or "the" removed. Did you forget to commit?

**Comment by JasonGross** [comment]
I removed the "the" in "does not implement the ``no associativity``" and the "a" in "replaces it by a ``left associativity``"

**Comment by Zimmi48** [comment]
Ah right!

**Review by Zimmi48 — APPROVED** [review]

**Pull request merged**